model-setup-examples/basic-elevon.md

```diff
@@ -0,0 +1,22 @@
+## Basic Elevon
+
+Elevons are control surfaces that combine the functions of the elevator and the aileron.
```

Comment:
My thoughts are that we either know or don't know what a flying wing is... Are we to expand the scope of the manual to covering those basics as well?